examples/resources/random_string/import.sh

```diff
@@ -1,2 +1,50 @@
-# Strings can be imported by just specifying the value of the string:
+# Random String can be imported by specifying the value of the string.
+#
+# "**NOTE**: Any attribute values that are specified within TF config will be
+# ignored during import and all attributes that have defaults defined within
+# the schema will have the default assigned.
+#
+# For instance, using the following config during import:
+# ```
+# resource "random_string" "test" {
+#   length = 16
+#   lower  = false
+# }
+# ```
+#
+# Then importing the resource using:
+# `terraform import random_string.test test`
+#
+# Would result in the triggering of a replacement (i.e., destroy-create) during
+# the next `terraform apply`. The replacement can be avoided by either using:
+#
+# 1. Attribute values that match the imported ID and defaults:
+# ```
+# resource "random_string" "test" {
+#   length = 4
+#   lower  = true
+# }
+# ```
+#
+# 2. Attribute values that match the imported ID and omit the attributes with defaults:
+# ```
+# resource "random_string" "test" {
+#   length = 4
+# }
+# ```
+#
+# 3. `ignore_changes` specifying the attributes to ignore:
+# ```
+# resource "random_string" "test" {
+#   length = 16
+#   lower  = false
+#
+#   lifecycle {
+#     ignore_changes = [
+#       length,
+#       lower,
+#     ]
+#   }
+# }
+# ```
```

Comment:
With the additional documentation that we've added to the `string.md.tmpl` we can remove the extra comments that were added to `import.sh`.

```suggestion
# Random String can be imported by specifying the value of the string. 
```